REQUIREMENTS.txt

```diff
@@ -10,7 +10,7 @@ scikit-image>=0.14.0
 scikit-learn
 scipy
 showit >= 1.1.4
-slicedimage == 0.0.1
+slicedimage == 0.0.3
```

Comment:
Shouldn't this be 0.0.2?

Reply:
Nope, found some issues in 0.0.2 that necessitated some bugfixes.